releasenotes/notes/securitycontext-on-auto-instrumentation-init-container-302976a9bf1bb629.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Configure SecurityContext on APM auto-instrumentation init container to
+    ensure container runs as the intended user.
```

Comment:
```suggestion
    Configure the SecurityContext on the APM auto-instrumentation init container to
    ensure the container runs as the intended user.
```

Reply:
Thanks @urseberry have implemented this change.